lmp/preload-app-images

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import argparse
+import logging
+import json
+import os
+import subprocess
+
+
+logging.basicConfig(level='INFO')
+fh = logging.FileHandler('/archive/preload-app-images.log')
+fh.setFormatter(logging.getLogger().handlers[0].formatter)
+logging.getLogger().addHandler(fh)
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('installed_versions')
+    parser.add_argument('dst_dir')
+    parser.add_argument('arch')
+    return parser.parse_args()
+
+
+def preload_app_images(target_json_file_filepath, dst_dir, arch):
+    with open(target_json_file_filepath) as target_json_file:
+        target_json = json.load(target_json_file)
+
+    target_name = list(target_json.keys())[0]
+    logging.info('Preloading app images for ' + target_name)
+    sha = target_json[target_name]['custom']['containers-sha']
+    image_data_dir = os.path.join('/var/cache/bitbake/app-images/', sha)
```

Comment:
I guess this directory will grow over time. I've added a note in Jira about this so we remember to attempt to prune old data eventually: https://foundriesio.atlassian.net/jira/software/projects/FF/boards/13?selectedIssue=FF-11

Reply:
we might be able to reason about what to delete by looking at targets.json and understanding that no new LMP builds would use an existing tarball, so it could be safe to delete. then we'd basically always keep the minimal amout of tarballs around. but that can be added further down the road

Reply:
I consider it as a temporal, putting image data is simple and straightforward and good for a start, but it has many drawbacks.

- First of all, it requires the container and lmp builds to have access to the same NFS volume;
- secondary, this storage has size limits and is not redundant and so reliable as GCS or S3; 
- Once we move towards using ostree as a storage for container images we will have to store images somewhere on the cloud (most likely Treehub backed by S3 for the beginning) so devices have access to it.